governance/votes/Executive vote - July 25, 2024.md

```diff
@@ -0,0 +1,162 @@
+date: 2024-07-25T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] LITE-PSM-USDC-A Phase 1 Setup, Spark Proxy Spell - July 25, 2024
+
+The Governance Facilitators, Dewiz, Sidestream, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **actions** will occur within the Maker Protocol:
+
+- LITE-PSM-USDC-A Phase 1 actions:
+  - The LitePSM Facility will be onboarded.
+  - PSM-USDC-A [Debt Ceiling Instant Access Module (DC-IAM)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-debt-ceiling-instant-access-module-dc-iam-) parameters will be updated.
+  - LITE-PSM-USDC-A DC-IAM parameters will be set up.
+  - The LITE_PSM_MOM GSM Delay Exception will be activated.
+  - **20 million USDC** will be migrated from PSM-USDC-A to LITE-PSM-USDC-A.
+  - The [GSM Pause Delay](https://mips.makerdao.com/mips/details/MIP113#10-1-gsm-governance-security-module-pause-delay) will be decreased.
+  - The [Emergency Shutdown Module (ESM)](https://docs.makerdao.com/smart-contract-modules/shutdown/the-emergency-shutdown-process-for-multi-collateral-dai-mcd) [Minimum Threshold](https://docs.makerdao.com/smart-contract-modules/shutdown/emergency-shutdown-module) will be increased.
+  - A LitePSM Keeper Network Job will be added.
+  - Six additions will be made to the [Chainlog](https://chainlog.makerdao.com).
+- A Spark Proxy Spell will be triggered.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **30 hours**.
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### LITE-PSM-USDC-A Phase 1 Actions
```

Comment:
🟢 **nit, non-blocking**

Consider adding a link to the [technical overview](https://forum.makerdao.com/t/technical-overview-lightweight-peg-stability-module-dss-lite-psm/24730) now that it is available.